showCrime/showCrime/settings.py

```diff
@@ -1,3 +1,12 @@
+"""
+django app for https://OakCrime.org
+"""
+
+__author__ = "rik@electronicArtifacts.com"
+__credits__ = ["clinton.blackburn@gmail.com","actionspeakslouder@gmail.com"]
+__date__ = "190128"
```

Comment:
The date format is unknown. I'm guessing this is supposed to map to 2019-01-28 (today's date). It's unclear why the change date is important enough to warrant editing every time a file is changed. Git already keeps track of when file modification dates at a higher granularity—includes time. This information will _quickly_ grow stale as developers forget to change it. It's extra overhead to development that isn't necessary.